app/javascript/packs/webauthn-setup.js

```diff
@@ -1,19 +1,37 @@
-const WebAuthn = require('../app/webauthn');
+import { loadPolyfills } from '@18f/identity-polyfill';
+import { isWebAuthnEnabled, enrollWebauthnDevice } from '../app/webauthn';
+
+/**
+ * Reloads the current page, presenting the message corresponding to the given error key.
+ *
+ * @param {string} error Error key for which to show message.
+ */
+function reloadWithError(error) {
+  const params = new URLSearchParams(window.location.search);
+  if (params.get('error') !== error) {
+    params.set('error', error);
+    window.location.search = params.toString();
+  }
```

Comment:
Following-up on an [earlier comment](https://github.com/18F/identity-idp/pull/5632/files#r765801409), as I'm writing a spec here, it occurs to me that we may want to allow reloading with the same error in the case that server-side fails with the same error. But we still need to avoid the initial "not supported" error getting stuck in a redirect loop.

Reply:
> Following-up on an [earlier comment](https://github.com/18F/identity-idp/pull/5632/files#r765801409), as I'm writing a spec here, it occurs to me that we may want to allow reloading with the same error in the case that server-side fails with the same error. But we still need to avoid the initial "not supported" error getting stuck in a reload loop.

Addressed in d498b34.